bin/serve.js

```diff
@@ -370,6 +370,7 @@ const loadConfig = async (cwd, entry, args) => {
 			'--no-etag': Boolean,
 			'--symlinks': Boolean,
 			'--cors': Boolean,
+			'--no-port-switching': Boolean,
```

Comment:
Could you add documentation for this?

Reply:
I've added it to --help. Would you like to have it added somewhere else as well?

Reply:
True, that will probably suffice. Thank you.